lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -1145,6 +1148,20 @@ namespace Js
             return proxy->PropertyKeysTrap(JavascriptProxy::KeysTrapKind::GetOwnPropertyNamesKind, scriptContext);
         }
 
+        if (ModuleNamespace::Is(object))
+        {
+            ModuleNamespace *ns = ModuleNamespace::FromVar(object);
+            JavascriptArray *names = ns->GetSortedExportedNames();
+            if (names == nullptr)
+            {
+                names = JavascriptObject::CreateOwnStringPropertiesHelper(object, scriptContext);
+                names->Sort(nullptr);
+                ns->SetSortedExportedNames(names);
```

Comment:
Can we actually cache this?  The array is returned to the user and it is mutable right?